tarodb/asset_minting.go

```diff
@@ -256,12 +256,14 @@ func (a *AssetMintingStore) CommitMintingBatch(ctx context.Context,
 		// internal key inserted above, we can create the set of new
 		// seedlings.
 		for _, seedling := range newBatch.Seedlings {
+			groupKey := asset.ToSerialized(&seedling.GroupKey)
 			err := q.InsertAssetSeedling(ctx, AssetSeedlingShell{
 				BatchID:         batchID,
 				AssetName:       seedling.AssetName,
 				AssetType:       int16(seedling.AssetType),
 				AssetSupply:     int64(seedling.Amount),
 				AssetMeta:       seedling.Metadata,
+				GroupKey:        groupKey.CopyBytes(),
```

Comment:
nit: can just use `[:]` notation to avoid the additional copy, here and below.

Reply:
Or in fact just use `seedling.GroupKey.SerializeCompressed()` directly.
Though is this key always set? Or can it be a zero key? We should probably check it here and not write an all-zero public key to the DB (which should also save us some storage space over time).